include/deal.II/lac/la_vector.h

```diff
@@ -110,81 +110,88 @@ namespace LinearAlgebra
     /**
      * Multiply the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator*= (const Number factor);
+    virtual Vector<Number> &operator*= (const Number factor) override;
```

Comment:
I said this was not ready to merge but a request for comment. I should have added a tag sorry.